### PR TITLE
add zTree.v3 v3.5.22

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/zTree/zTree_v3
**Watch 103
  Star 791
 Fork 308**
@Amomo could u help me check this PR for #6816 ? thank you!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

# Essential checklist
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update) https://github.com/zTree/zTree_v3/issues/129
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Do we need `ajax/libs/zTree.v3/3.5.21/css/demo.css` and `ajax/libs/zTree.v3/3.5.21/api`?

**Comment by maruilian11** [comment]
@PeterDaveHello sorry, we don't need them.
@Amomo could you help me check this PR? thanks!

File: ajax/libs/zTree.v3/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/zTree/zTree_v3.git",
+    "basePath": "",
+    "files": [
+      "css/zTreeStyle/zTreeStyle.css",
+      "css/zTreeStyle/img/zTreeStandard.+(gif|png)",
+      "css/zTreeStyle/img/loading.gif",
+      "css/zTreeStyle/img/line_conn.gif",
+      "css/zTreeStyle/img/diy/.png",
```
Comment (Amomo) [inline ajax/libs/zTree.v3/package.json]: maybe `*.png` ?

File: ajax/libs/zTree.v3/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zTree/zTree_v3.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/zTree/zTree_v3.git",
+    "basePath": "",
+    "files": [
+      "css/zTreeStyle/zTreeStyle.css",
```
Comment (Amomo) [inline ajax/libs/zTree.v3/package.json]: `zTreeStyle*.css`?

**Comment by Stephan972** [comment]
Is zTree available on cdnjs? If so, how can I add it to my HTML page. I can't seem to find it on https://cdnjs.com/libraries.

**Comment by PeterDaveHello** [comment]
Ping @maruilian11

**Comment by maruilian11** [comment]
@Amomo i have updated to v3.5.22, could you help me check it again? thanks!

File: ajax/libs/zTree.v3/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      "css/zTreeStyle/img/zTreeStandard.+(gif|png)",
+      "css/zTreeStyle/img/loading.gif",
+      "css/zTreeStyle/img/line_conn.gif",
+      "css/zTreeStyle/img/diy/*.png",
+      "css/metroStyle/metroStyle*.css",
+      "css/metroStyle/img/line_conn.png",
+      "css/metroStyle/img/loading.gif",
+      "css/metroStyle/img/metro.+(gif|png)",
+      "css/awesomeStyle/awesome*.css",
+      "css/awesomeStyle/img/loading.gif",
```
Comment (PeterDaveHello) [inline ajax/libs/zTree.v3/package.json]: Let's try `"css/*Style/**/*"` here.

**Pull request merged**